qubesadmin/tests/mock_app.py

```diff
@@ -648,6 +648,12 @@ def __init__(self):
         # # this system has some reasonable defaults
         self._qubes['dom0'].default_dispvm = "default-dvm"
 
+        self._qubes['test-vm2'] = MockQube(name="test-vm2", qapp=self,
+                                           features={'menu-favorites': ''})
+        self._qubes['dom0'].features[INITIAL_PAGE_FEATURE] = 'favorites_page'
+        self._qubes['dom0'].features[SORT_RUNNING_FEATURE] = '1'
+        self._qubes['dom0'].features[POSITION_FEATURE] = 'mouse'
```

Comment:
Those three are not defined in this file - either define them earlier in the file, or use literal values.